packages/app/src/files-manager/components/files-summary-item/files-summary-item.component.tsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React from "react"
+import Text, {
+  TextDisplayStyle,
+} from "Renderer/components/core/text/text.component"
+import { DiskSpaceCategory } from "App/files-manager/components/files-manager/files-manager.interface"
+import { backgroundColor } from "Renderer/styles/theming/theme-getters"
+import { FunctionComponent } from "Renderer/types/function-component.interface"
+import styled from "styled-components"
+import Icon, { IconSize } from "Renderer/components/core/icon/icon.component"
+import { FilesSummaryItemTestIds } from "App/files-manager/components/files-summary-item/files-summary-item-test-ids.enum"
+import { displaySpace } from "App/files-manager/helpers/display-space"
+
+const FilesSummaryItemContainer = styled.div`
+  display: flex;
+  align-items: flex-start;
+  background-color: ${backgroundColor("row")};
+  padding: 1.6rem;
+  width: 28rem;
+`
+
+const SummaryIcon = styled(Icon)<{ fileColor: string }>`
+  margin-right: 1.6rem;
+  background-color: ${({ fileColor }) => fileColor};
+  border-radius: 0.4rem;
+  padding: 1.2rem;
+`
+
+const TextWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 100%;
+`
+
+const FilesSummaryItem: FunctionComponent<DiskSpaceCategory> = ({
+  color,
+  filesAmount,
+  filesType,
+  megabyteSize = 0,
+  icon,
+}) => {
+  return (
+    <FilesSummaryItemContainer data-testid={FilesSummaryItemTestIds.Wrapper}>
+      <SummaryIcon fileColor={color} type={icon} size={IconSize.Medium} />
+      <TextWrapper>
+        <Text
+          displayStyle={TextDisplayStyle.LargeText}
+          element={"p"}
+          data-testid={FilesSummaryItemTestIds.Title}
+        >
+          {filesType}
+        </Text>
+        <Text
+          displayStyle={TextDisplayStyle.MediumFadedText}
+          element={"p"}
+          data-testid={FilesSummaryItemTestIds.Description}
+        >
+          {filesAmount
+            ? `${filesAmount} files (${displaySpace(megabyteSize)})`
```

Comment:
🟡
`megabajt` or `mebibajt` we should avoid mixing units. Certainly in the view layer.

I suggest to in the whole application keep values in bytes and only in transforming functions converted to view units.

Reply:
I moved this discussion to slack channel 👉  https://appnroll.slack.com/archives/C027FSM1VPZ/p1643183400004900

Reply:
memorySpace changed to bytes in [ce40a64](https://github.com/mudita/mudita-center/pull/598/commits/ce40a6429870732cb157448924ac3de654ac19c3)